unity/Holo/Assets/Editor/InputConfiguration.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.IO;
+using UnityEngine;
+using UnityEditor;
+
+using NDesk.Options;
+
+class InputConfiguration
+{
+    private static readonly log4net.ILog Log = log4net.LogManager.GetLogger(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType);
+
+    public string RootDirectory { get; private set; } = null;
+    public string OutputDir { get; private set; } = null;
+    public string LogFileDir { get; private set; } = null;
+    private OptionSet options = new OptionSet();
+
+    public InputConfiguration()
+    {
+        if (Application.isBatchMode)
+        {
+            GetBatchModeArgs();
+        }
+        else
+        {
+            RootDirectory = EditorUtility.OpenFolderPanel("Select model root folder with ModelInfo.json", Application.dataPath, "");
+        }
+        if (string.IsNullOrEmpty(OutputDir))
+        {

```

Comment:
Log indicating that OutpuDir is empty and we are defaulting to following path would be nice

Reply:
Done.